### DISCUSSION TITLE
Is that possible to make data migrations with Prisma?

### DISCUSSION THREAD

**Discussion by abriginets** [prisma/prisma#8933] [Q&A]
Hello. I'm trying out Prisma with PlanetScale MySQL and I'm wondering if that's possible to not only migrate schema changes but also migrate data? For example, I would like to hit an API to fetch airports data and then prepare a migration to fill out the `airports` table in my database with this data. At where I work we have Knex using which we can literally write JavaScript code and prepare ORM-based statements to run before deployment which really saves a lot of time.

**Comment by ryands17** [comment]
@abriginets 👋 

Currently we do not support data migrations but we do have a feature request that you can add a 👍 on and add your use case 🙂 
https://github.com/prisma/prisma/issues/6345

**Comment by janpio** [comment]
Note: PlanetScale only copies schema changes between branches. So if you are using Prisma and PlanetScale together with their branching system, you can _not_ modify data via deploy requests right now.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
